systems/subscriber/api-gateway/pkg/rest/router_test.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/gin-gonic/gin"
```

Comment:
These commonly used dummy values:
```
                                        Iccid:          "1234567890123456789",
					SimType:        "ukama_data",
					Msisdn:         "555-555-1234",
					SmDpAddress:    "http://example.com",
					ActivationCode: "abc123",
					QrCode:         "qr123",
					and more ....
                                        and more ....
                                        and more ....
```
should be defined as **const** and reused throughout tests cases with minor variations, instead of having them hardcoded and duplicated overall in these tests cases.

Reply:
addressed